typescript/lib/services/service_context.ts

```diff
@@ -0,0 +1,125 @@
+import * as bindingsJsGrpc from '../../kurtosis_core_rpc_api_bindings/api_container_service_grpc_pb'; //TODO - importing all modules or just specific ones
+import * as bindingsJs from '../../kurtosis_core_rpc_api_bindings/api_container_service_pb'; //TODO (comment) - I cannont merge with above line unfortunately, can only import one module at a time MAXIMUM
+import { ServiceID } from './service';
+import { StaticFileID } from './container_creation_config'; 
+import * as grpc from "grpc";
+import * as constructors from "../constructor_calls";
+var path = require("path"); //TODO - I don't if this works
```

Comment:
My code compiles with this import statement, but I am very unsure whether or not it works. I just want a way to concatenate two paths together, and this path requires a node installation as well (which I haven't installed). Perhaps, I should just concatenate two paths manually, but it might be a lot of work. Hard to test your code when you don't know if your implementation causes an error or not (I will figure this out once I start porting the testsuite)